packages/core/src/routes/admin-user.ts

```diff
@@ -110,7 +110,7 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
       params: object({ userId: string() }),
       body: object({
         name: string().nullable().optional(),
-        avatar: string().url().nullable().optional(),
+        avatar: string().url().nullable().optional().or(literal('')),
```

Comment:
feels like the order should be. string().url().or(literal('')).nullable().optional(). Besides if it is a common case we should have some utils guard created like `allowEmpty`

Reply:
Good call, updated.
As for the `allowEmpty` util, I prefer leave it to LOG-3145.